static/js/exclusive-content.js

```diff
@@ -0,0 +1,139 @@
+const WORKER = 'https://webmonetization.org/api/exclusive-content'
```

Comment:
Should this new static file be added to the [docusaurus config](https://github.com/WICG/webmonetization/blob/master/docusaurus.config.js#L183) or [README](https://github.com/WICG/webmonetization#directory-structure)?

Reply:
I don't think it needs to go into the docusaurus config since that says `Add custom scripts here that would be placed in <script> tags.` and we don't need this file to be loaded into a script tag. But it probably should go into the README, together with the worker.

Reply:
Added it to the README